package/gluon-mesh-vpn-tunneldigger/luasrc/lib/gluon/upgrade/400-mesh-vpn-tunneldigger

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site_config'
+local users = require 'gluon.users'
+local util = require 'gluon.util'
+
+local uci = require('simple-uci').cursor()
+
+
+-- Group for iptables rule
+users.add_group('gluon-tunneldigger', 900)
+
+local enabled = uci:get_first('tunneldigger', 'broker', 'enabled')
+if (tonumber(enabled) == 0) then
+  enabled = site.tunneldigger_mesh_vpn.enabled and 1 or 0
+end
+
+-- Delete old broker config section       
+uci:delete_all('tunneldigger', 'broker')
+
+section = uci:add('tunneldigger', 'broker')
+
+uci:section('tunneldigger', 'broker', section,
```

Comment:
It would be nicer to just explicitly name the section instead of using an anonymous section. You could just 
```lua
uci:delete('tunneldigger', 'mesh_vpn')
uci:section('tunneldigger', 'broker', 'mesh_vpn', ...)
```
instead of deleting all broker sections. Instead of `uci:get_first` you could use `uci:get_bool`, like we to in other scripts.

For migration from an older configuration using an anonymous section, you could use:
```lua
if not uci:get('tunneldigger', 'mesh_vpn') then
	uci:delete_all('tunneldigger', 'broker')
end
```